packages/neuron-wallet/src/services/tx/transaction-generator.ts

```diff
@@ -124,21 +127,24 @@ export class TransactionGenerator {
       .map(input => BigInt(input.capacity))
       .reduce((result, c) => result + c, BigInt(0))
 
-    const minCellCapacity = BigInt(MIN_CELL_CAPACITY)
     const outputs: Output[] = targetOutputs.map((o, index) => {
       const { capacity, address } = o
 
-      // skip last output
-      if (BigInt(capacity) < minCellCapacity && index !== targetOutputs.length - 1) {
-        throw new CapacityTooSmall()
+      let lockScript: Script | undefined
+      if (FullAddress.isFullFormat(address)) {
+        lockScript = FullAddress.parse(address)
+      } else {
+        const blake160: string = LockUtils.addressToBlake160(address)
+        lockScript = new Script(codeHash, blake160, hashType)
```

Comment:
Can we extract this to something as `parseAddressToScript`?

Reply:
ok

Reply:
Updated, using `AddressParser` to parse all addresses types